hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/freon/TestOmBucketReadWriteFileOps.java

```diff
@@ -293,22 +293,11 @@ private ParameterBuilder setBufferSize(int bufferSizeParam) {
       return this;
     }
 
-    private ParameterBuilder setLength(int lengthParam) {
-      length = lengthParam;
-      return this;
-    }
-
     private ParameterBuilder setTotalThreadCount(int totalThreadCountParam) {
       totalThreadCount = totalThreadCountParam;
       return this;
     }
 
-    private ParameterBuilder setReadThreadPercentage(
-        int readThreadPercentageParam) {
-      readThreadPercentage = readThreadPercentageParam;
-      return this;
-    }
```

Comment:
These are no longer unused with #7870: `TestOmBucketReadWriteFileOps` and `TestOmBucketReadWriteKeyOps` are refactored to use the same `ParameterBuilder` class (get rid of duplication) and test cases tweaked to exercise each parameter.

Reply:
Thanks, but the suppression will cause conflict in the other PR.  I'd prefer to merge that one first, since it reduces CI time and was opened 11 days ago.